src/server.tsx

```diff
@@ -5,19 +5,24 @@ import React from 'react';
 import { renderToStaticMarkup } from 'react-dom/server';
 import { extractCritical } from 'emotion-server';
 import { renderHtmlDocument } from './utils/renderHtmlDocument';
-import { fetchDefaultEpicContent, DefaultEpicContent } from './api/contributionsApi';
+import { fetchDefaultEpicContent } from './api/contributionsApi';
 import { DefaultEpic } from './components/DefaultEpic';
 import cors from 'cors';
 
-const bootApp = (content: DefaultEpicContent): void => {
-    const app = express();
-    app.use(express.json({ limit: '50mb' }));
+// Pre-cache API response
+fetchDefaultEpicContent();
```

Comment:
This is the line I mentioned in the PR description. Happy to remove this if we don't think it's worth it.

Reply:
Seems worthwhile!